.github/workflows/version.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump Versions
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  version-packages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          cache: yarn
+          node-version: v20.7.0
+
+      - name: node_modules cache
+        uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-node-modules-cache-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-node-modules-cache-
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Bump versions
+        run: yarn run lerna-version --yes
+
+      - name: Commit version changes
+        run: |
+          git add yarn.lock
+          git add '*.json'
+          git commit -m "build: bump versions"
+
+      - name: Commit CHANGELOG changes
+        run: |
+          yarn prettier-write
+          git add '*.md'
+          git commit -m "docs: update CHANGELOGs"
```

Comment:
Do you need the lowercase "s" or are you attempting to pluralize "CHANGELOG"?

Reply:
It updates `CHANGELOG` in every package and the root package, so it's multiple of them.